src/components/pages/SearchResultsPage.js

```diff
@@ -1,88 +1,264 @@
-import { React, useState } from "react";
+import { React, useState, useEffect, useContext, useCallback } from "react";
 import { useLocation } from "react-router-dom";
 import Navbar from "../reusable/NavBar";
 import Sidebar from "../reusable/SideBar";
 import Graph from "../reusable/Graph";
-import StockTable from "../singular/StockTable";
+import AlgoquantApiContext from "../../api/ApiContext";
+import Table from "../reusable/Table";
+import { SaveSpinner } from "../reusable/LoadSpinner";
+import { GraphSpinner } from "../reusable/LoadSpinner";
 import GraphStats from "../reusable/GraphStats";
+import { filters } from "../utils/filtersEnum";
 
 const SearchResultsPage = () => {
   const location = useLocation();
+  // State variables used to access algoquant SDK API and display/ keep state of user data from database
+  const algoquantApi = useContext(AlgoquantApiContext);
 
-  // Currently hardcoded but will eventually come from API
-  const [chartData, setChartData] = useState([
-    50, 41, 35, 51, 4, 62, 262, 91, 134,
-  ]);
-
-  const [categories, setCategories] = useState([
-    "Jan",
-    "Feb",
-    "Mar",
-    "Apr",
-    "May",
-    "Jun",
-    "Jul",
-    "Aug",
-    "Sep",
-  ]);
-
-  const [stockData, setStockData] = useState([
+  // State variable to keep track of what timeframe of data is fetched
+  const [selectedFilter, setSelectedFilter] = useState("Today");
```

Comment:
The state should be `Filters.day` , using the filters enum every would make it easier for us to later change filters.

Reply:
fixed